pkg/api/pod/util.go

```diff
@@ -329,6 +329,22 @@ func usesHugePagesInProjectedEnv(item api.Container) bool {
 	return false
 }
 
+// hasSysctlsWithSlashNames return value is determined by the following:
+// 1.use strict validation for Create requests, and for Update requests where the existing object's sysctl does not contain a forward slash, return false
```

Comment:
does this comment belong to the function?

Reply:
The description has been modified. Thanks.